src/app/src/mainwindow.h

```diff
@@ -217,15 +217,14 @@ class MainWindow : public QMainWindow {
             .bufferCount = 100
         };
 
-    private:
```

Comment:
can you put this back, it's a cpp style thing to separate methods from variables for the different access classes